src/prng.rs

```diff
@@ -61,7 +61,7 @@ fn random_field_from_seed(seed: &[u8], length: usize) -> Vec<Field> {
         cipher.apply_keystream(&mut buffer);
 
         // rejection sampling
-        for chunk in buffer.chunks_exact(std::mem::size_of::<Field>()) {
+        for chunk in buffer.chunks_exact(4) {
```

Comment:
Same comment as in `src/util.rs`: let's put this back to `size_of` if we can.

Reply:
Oh, is there a trait for this that `Fp` could implement? (See comment above.)

Reply:
Resolving this, since size_of doesn't seem appropriate here.